application/jaxrs-mongodb-quarkus/src/test/java/webapp/tier/service/MongodbServiceTest.java

```diff
@@ -1,45 +1,93 @@
 package webapp.tier.service;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.when;
 
+import java.security.NoSuchAlgorithmException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.bson.Document;
 import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoCursor;
+
+import io.quarkus.test.junit.QuarkusTest;
+import webapp.tier.bean.MsgBean;
 
+@QuarkusTest
 class MongodbServiceTest {
 
+	@Inject
+	MongodbService mongosvc;
+
+	String respbody = "message: Hello k8s-3tier-webapp with quarkus";
+
 	@Test
-	void testInsertMysqlError() {
-		MongodbService svc = new MongodbService();
-		try {
-			svc.insertMsg();
-			fail();
-		} catch (Exception e) {
-			e.printStackTrace();
-			assertEquals("Insert Error.", e.getMessage());
-		}
+	@SuppressWarnings("unchecked")
+	void testInsertMysql() throws NoSuchAlgorithmException {
+		MongoCollection<Document> col = Mockito.mock(MongoCollection.class);
+		when(col.insertOne(ArgumentMatchers.any())).thenReturn(null);
+		assertThat(mongosvc.insertMsg(col).getFullmsg(), containsString(respbody));
 	}
 
 	@Test
-	void testSelectMsgError() {
-		MongodbService svc = new MongodbService();
-		try {
-			svc.selectMsg();
-			fail();
-		} catch (Exception e) {
-			e.printStackTrace();
-			assertEquals("Select Error.", e.getMessage());
-		}
+	@SuppressWarnings("unchecked")
+	void testSelectMsgIsEmpty() {
+		MongoCursor<Document> cursor = Mockito.mock(MongoCursor.class);
+		FindIterable<Document> ite = Mockito.mock(FindIterable.class);
+		MongoCollection<Document> col = Mockito.mock(MongoCollection.class);
+		when(col.find()).thenReturn(ite);
+		when(ite.iterator()).thenReturn(cursor);
+		mongosvc.selectMsg(col);
+	}
+
+	@Test
+	@SuppressWarnings("unchecked")
+	void testSelectMsgWithMessage() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5923183)